cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -577,22 +618,11 @@ func main() {
 
 	if *listen != "" {
 		go func() {
-			trace.AuthRequest = func(req *http.Request) (any, sensitive bool) {
-				return true, true
-			}
-			prom := promhttp.Handler()
-			h := func(w http.ResponseWriter, r *http.Request) {
-				switch r.URL.Path {
-				case "/debug/requests":
-					trace.Traces(w, r)
-				case "/metrics":
-					prom.ServeHTTP(w, r)
-				default:
-					s.ServeHTTP(w, r)
-				}
-			}
+			pp := http.NewServeMux()
```

Comment:
nit: rename `pp` to something more clear. Is it `prometheusProxy`? or?

Reply:
Could just use `mux`

Reply:
yep, lazy copy paste from sourcegraph/sourcegraph